hibp/client.go

```diff
@@ -62,3 +62,31 @@ func createClient(ctx context.Context, apiKey string) *hibp.Client {
 	cl := hibp.New(clientOptions...)
 	return &cl
 }
+
+// Create a client without an API key
+// Certain tables like hibp_password do not require an API key
+func getHibpClientWithoutAPIKey(ctx context.Context, d *plugin.QueryData) (*hibp.Client, error) {
```

Comment:
Do we need a separate function for this? I think we can use the existing one, and if we don't find any keys in the config or environment variables, we create a client without the key.